tools/bundle_adjust_tracks.cxx

```diff
@@ -180,9 +180,18 @@ static kwiver::vital::config_block_sptr default_config()
                     "when updating cameras. This option is only relevent if a "
                     "value is give to the input_pos_files option.");
 
-
       config->set_value("krtd_clean_up", "false",
                         "Delete all previously existing KRTD files present in output_krtd_dir before writing new KRTD files.");
+
+  config->set_value("depthmaps_points_file", "",
```

Comment:
It looks like `depthmaps_images_file` is the only config option that is actually used by the GUI in this branch,  so we should be able to remove the other unused ones (`depthmaps_surfaces_file` and `depthmaps_points_file`) right?

This actually is not the right place for the `depthmaps_images_file` option either because it is not used by the bundle adjustment tool.  Ideally there would be another MAP-Tk tool that replaces the `cudaPlanesweepMAPTK` and uses a Vital configuration file for its settings like the rest of these tools.  That tool would include `depthmaps_images_file` as well as all the other parameters needed (path to cameras, image file list, etc.).  But we don't have that yet.

Maybe the better approach is to create a template for a GUI config files that contains only the config options that are loaded by the GUI along with comments describing what the options do.  We could put this file in the `config` or maybe `examples` directory so it would be installed with MAP-Tk.  Then users could use this template to manually provide what is needed to load all the content for their project.